### ISSUE TITLE
`scan failed to produce proper report` for repos without vulnerabilities

### ISSUE THREAD

**Issue by Akaame** [Templum/govulncheck-action#11]
```
govulncheck ./... 
govulncheck is an experimental tool. Share feedback at https://go.dev/s/govulncheck-feedback.

Scanning for dependencies with known vulnerabilities...
No vulnerabilities found.
```

For this repo with no vulnerabilities the following error is printed out from the action:

 `ERR Scanning yielded error error="scan failed to produce proper report".`
 
```yaml
steps:
      - uses: actions/checkout@v3
      - uses: Templum/govulncheck-action@v0.0.5
```

Would it be possible to check the action behavior for a clean repo?

**Comment by Templum** [comment]
I'm using the action with this repository which has no vulnerability. Or what did you mean by a clean repository?

**Comment by Templum** [comment]
The error itself only shows up, when the json produced by govulncheck is not parseable, so it's likely that you misconfigured the path

**Comment by Templum** [comment]
Could you please double-check the logs and maybe attach them, with the given details I sadly can't do much investigating.

**Comment by Templum** [comment]
If you use the `@main` version of this action it should now also include printing of the encountered error. Maybe that allows us to narrow down the origin.

**Comment by Templum** [comment]
Probably the issue is linked to the fact that the JSON Report Format was drastically changed. This should be fixed by #15

**Comment by Templum** [comment]
If issue still occurs and is not fixed after new release, feel free to create a new Issue.
